src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -957,11 +960,43 @@ table.logs .metadata {
 }
 table.logs tr td:last-child { width: 100%; }
 
-.log-levels { width: 100%; margin: 0; padding: 0; display: flex; align-items: center; list-style: none; }
-.log-levels li { width: 100%; padding: 3px; margin: 0; cursor: pointer; text-align: center; border: 2px dashed #e0e0e0; border-radius: 5px; color: #888; }
-.log-levels li + li { margin-left: 10px; }
-.log-levels li.active { background: #eee; color: #666; border-style: solid; border-width: 1px; padding: 4px; border-color: #aaa; }
-.log-levels li.last-active { cursor: not-allowed; }
+.log-levels { position: absolute; right: 5px; top: 33px; }
+.log-levels { border: var(--border); box-shadow: var(--shadow); margin: 0; padding: 0; display: flex; flex-direction: column; align-items: flex-end; }
+.log-levels:before {
+    content: "Filter";
```

Comment:
perhaps `Level` is better (given we already know it's filter by icon), that would enable a second `Channel` filter as well (which was on my list). Unless you want to combine things here of course :)

Reply:
We can still change things in 4.2, the 2 month period is to tweak things that are already part of the release, which is the case here.

Reply:
Yes ... but in my opinion what Roland is proposing (table multi-filtering) would require lots of changes. @ro0NL if you think you can do it in time for 4.2, please close this PR and implement the multi-filtering inside the table columns. Thanks 🙏 !

Reply:
i thought it could share the same JS basically, but that's not the case (yet).

Channel filters doesnt need the level approach (selecting all higher levels). So yeah, maybe that's something for 4.3, we'll see :)

Still, using `Levels` for a label makes sense to let the user know what this filter is about ;)

Reply:
If that's too much work, it then makes sense to improve what we have for 4.2 and do more/better in 4.3.

Reply:
I've discussed about this with @ro0NL on Symfony Slack. We agree to leave this "as is" for Symfony 4.2 (so this PR is OK to be merged).

In Symfony 4.3 we strive to create a multi-purpose filter logic to add filters to all/most column headers of all/most tables. Thanks!